tuxemon/resources/db/locale/eo.json

```diff
@@ -5,18 +5,18 @@
     "combat_used_x_on_y": "${{name}} estis uzata al ${{target}} de ${{user}}!",
     "combat_call_tuxemon": "Iru, ${{name}}!",
     "combat_opponent_call_tuxemon": "${{user}} sendas ${{name}}!",
-    "combat_state_poison_damage": "${{name}} estis venenigita!",
-    "combat_status_damage": "${{name}} estis damaĝita de ${{status}}!",
+    "combat_state_poison_damage": "${{name}} veneniĝis!",
+    "combat_status_damage": "${{name}} damaĝiĝis pro ${{status}}!",
     "combat_victory": "Vi gajnis!",
     "combat_defeat": "Vi estis venkita!",
-    "combat_capturing success": "Vi kaptis ${{name}}!",
+    "combat_capturing success": "${{name}} estias kaptita!",
     "combat_capturing_fail": "${{name}} liberiĝis!",
     "combat_replacement": "Elektu anstataŭanton!",
     "combat_replacement_is_fainted": "La monstro svenis!",
     "combat_wild_appeared": "Sovaĝa ${{name}} aperis!",
-    "combat_monster_choice": "Kion ${{name}} faris?",
+    "combat_monster_choice": "Kion ${{name}} faru?",
     "combat_fainted": "${{name}} svenas!",
```

Comment:
This string is not present, is imperative. I chose to translate it in imperative form as IMHO gives the nuance about the order.
The [original one](https://github.com/Tuxemon/Tuxemon/blob/development/tuxemon/resources/db/locale/en_US.json#L17) is in future form, not in the past on. 
Obviously it doesn't have sense to translate in past form: "What NameOfTuxemon did?" but it already didn't do anything. LOL
I think that the imperative is correct but if you have a proposal, write it down there and let's discuss ;)